node/src/runner.rs

```diff
@@ -71,13 +71,18 @@ pub async fn initialize(
     });
 
     // P2P subsystem
+    // TODO: Replace Lmdb with Sqlite backend when it's read
```

Comment:
ready

Reply:
Thanks, I will fix it next time (don't want for CI).